backend/config/config.py

```diff
@@ -102,8 +104,11 @@ def _load_config(path=None):
     'geocoding': True,
     'address_database': True,
     'geocoder': 'oxcoder',
-    'measurer': 'haversine',
-    'number_of_adequacy_processors': 8,
+    'measurement': {
```

Comment:
Why this method rather than an additional layer in measurement?

measurement: {
    haversine: {executor, processor},
    osrm: {executor, processor}
}

Reply:
That looks good, but how do we get the measurer?

Reply:
When we can have:

`measurer`: `haversine` ,
`measurer_config` : {
    haversine: {executor, processor},
    osrm: {executor, processor}
}